src/main/java/com/aws/greengrass/telemetry/TelemetryAgent.java

```diff
@@ -142,11 +142,18 @@ public TelemetryAgent(Topics topics, MqttClient mqttClient, DeviceConfiguration
         getPeriodicAggregateTimeTopic();
         getPeriodicPublishTimeTopic();
         schedulePeriodicAggregateMetrics(false);
```

Comment:
why not move L144 also? We don't need to schedule right?

Reply:
It can be helpful to keep the periodic aggregation. Even though it can't be uploaded, may still be useful offline?

Reply:
OK.